### ISSUE TITLE
Q-Q plot expectations

### ISSUE THREAD

**Issue by jdstamp** [jdstamp/leiden_paper#5]
As discussed on PR #3 , we should consider plotting the Q-Q plots in a different manner.

TLDR;
The GWAS literature primarily scales Q-Q plots in a way that the expectation is `x=y`. The Q-Q plot references don't set this expectation for Q-Q plots and as such the R package for plotting Q-Q plots using `ggplot2` computes the expectation from data quantiles and the rate parameter.

Do we want to follow GWAS convention and implement the plotting in a different way such that `x=y`? Do reviewers expect this?

**Comment by rbyrne5** [comment]
I've added a branch with an updated qqplot script that should produce a 1:1 line based on code from this blogpost:
https://danielroelfs.com/blog/how-i-make-qq-plots-using-ggplot/

The code is a little clunky and would not allow for easy changes to our parameter sets though so it might need a bit of work

**Comment by jdstamp** [comment]
Maybe something like 

```
df %>%
  group_by(adjusted, n_causal, heritability) %>%
  arrange(P, .by_group = TRUE) %>%
  mutate(
    cupper = cupper,
    clower = clower,
    expected= -log10(ppoints(n_snps)),
    observed = -log10(P)
  )
```

could work?

**Comment by rbyrne5** [comment]
Good catch, that seems to work perfectly for the small test dataset I ran it on and should be more flexible to different parameter sets. I will test it on the full dataset now to be sure

Assuming it works, let me know if you are happy to merge with main branch and possibly update the figure in the paper.

**Comment by jdstamp** [comment]
Awesome! Yes, I am ok with these changes. Just open a PR so I can give the green light there.
